plugins/polio/js/src/hooks/useGetCampaignHistory.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable camelcase */
+import moment from 'moment';
+import { UseQueryResult } from 'react-query';
+import { getRequest } from '../../../../../hat/assets/js/apps/Iaso/libs/Api';
+import { useSnackQuery } from '../../../../../hat/assets/js/apps/Iaso/libs/apiHooks';
+import { DropdownOptions } from '../../../../../hat/assets/js/apps/Iaso/types/utils';
+import { CampaignLogDetail, CampaignLogData } from '../constants/types';
+
+export const getCampaignLog = (
+    campaignId: string | undefined,
+): Promise<CampaignLogDetail> => {
+    return getRequest(
+        `/api/logs/?objectId=${campaignId}&contentType=polio.campaign`,
+    );
+};
+
+const getCampaignLogDetail = (
+    logId: string | undefined,
+): Promise<CampaignLogData> => {
+    return getRequest(`/api/logs/${logId}`);
+};
+
+export const useGetCampaignLogs = (
+    campaignId: string | undefined,
+): UseQueryResult<DropdownOptions<number>[], Error> => {
+    const queryKey: any[] = ['campaignLog', campaignId];
```

Comment:
this isn't really necessary, you could pass it directly in the params of `useSnackQuery`

Reply:
queryKey: ['campaignLog'] ? no need to add the parameter ?